model/flyExporter/extractor/QtiExtractor.php

```diff
@@ -447,15 +472,6 @@ public function getChoices($interaction)
                 $return['choice_label_' . $i] = ($choice) ?: '';
                 $i++;
             }
-            if ($this->headerChoice > count($return)) {
-                while ($this->headerChoice > count($return)) {
-                    $return['choice_identifier_' . $i] = '';
```

Comment:
this obviously was an attempt to fix subject issue, unfortunately it works only when the biggest number of choices is in the first interaction